stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -47,6 +48,10 @@ open class StripeEditText @JvmOverloads constructor(
     @ColorInt
     private var externalErrorColor: Int? = null
 
+    private val textWatchers by lazy {
+        mutableListOf<TextWatcher>()
+    }
```

Comment:
why does this need to be `by lazy`?

Reply:
I think it won't necessarily be there if no `TextWatcher` is added

Reply:
But taking another look `#listenForTextChanges` always register a textwatcher... removed the `lazy` delgate